test/screenshot/golden.json

```diff
@@ -556,6 +556,14 @@
       "desktop_windows_ie@11": "https://storage.googleapis.com/mdc-web-screenshot-tests/advorak/2018/09/21/16_50_45_748/spec/mdc-dialog/mixins/title-ink-color.html.windows_ie_11.png"
     }
   },
+  "spec/mdc-drawer/classes/dismissible-below-top-app-bar.html": {
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2018/10/31/20_39_25_363/spec/mdc-drawer/classes/dismissible-below-top-app-bar.html?utm_source=golden_json",
```

Comment:
Seems like the main content is not scrollable in this demo. Also the drawer is being clipped at the bottom when I resize the browser small viewport.

We need to update the docs too to layout main content and drawer correctly.